File: src/main/java/com/prog5/items/IDCard.java
package com.prog5.items;

public class IDCard {
    private String owner;
    private String cardNumber;

    private boolean isLost;

    public IDCard(String owner, String cardNumber, boolean isLost) {
        this.owner = owner;
        this.cardNumber = cardNumber;
        this.isLost = isLost;
    }

    @Override
    public String toString() {
        return "\nIDCard: " +
                "\nowner: " + owner +
                ", cardNumber: " + cardNumber +
                ", isLost: " + isLost;
    }
}

File: IDCard.cs
namespace Items
{
  public class IDCard
  {
    public string Name { get; set; }
    public string Number { get; set; }
    public bool Lost { get; set; }

    public IDCard(string name, string number, bool lost)
    {
        Name = name;
        Number = number;
        Lost = lost;
    }
  }
}
